core/chains/evm/headtracker/config.go

```diff
@@ -12,3 +16,31 @@ type Config interface {
 	EvmHeadTrackerMaxBufferSize() uint32
 	EvmHeadTrackerSamplingInterval() time.Duration
 }
+
+type EvmHtrkConfig types.HtrkConfig
```

Comment:
Isn't this just a `Config` too?
```suggestion
type Config types.Config
```

Reply:
thanks for the note. Will be refactoring that out as we already have headtracker context

Reply:
However i think that having `EVM` in the Config name could still be useful especially as that config is evm specific.

Reply:
> However i think that having `EVM` in the Config name could still be useful especially as that config is evm specific.

But is it used from any context that _isn't_ EVM specific?